client/src/GlobalStyles.js

```diff
@@ -42,11 +42,13 @@ const GlobalStyles = () => {
         }
         h1 {
           font-family: "SFUIbold";
+          letter-spacing: -0.05em;
```

Comment:
You can combine similar stylings to avoid adding it multiple times:

```
h1, h2, h3 {
  font-family: "SFUIbold";
  letter-spacing: -0.05em;
  color: #000000;
}

h1 {
  font-size: 2.2em;
}

h2 {
  font-size: 2em;
}

h3 {
  font-size: 1em;
}
```

Reply:
Very good point! Probably would've done it in a later step.

Reply:
Fix #48